double/internal/ryu/ryu.mbt

```diff
@@ -604,6 +576,9 @@ fn d2d_small_int(
 }
 
 pub fn ryu_to_string(val : Double) -> String {
+  if val == 0.0 {
```

Comment:
consider `-0.0`, it equals `0.0` but should be printed as `-0`

Reply:
I tried console, the JS engine does print it as "0"